client/render/gl_grass.cpp

```diff
@@ -1109,13 +1109,20 @@ R_GrassTextureForName
 find or add unique texture for grass
 ================
 */
-byte R_GrassTextureForName( const char *name )
+int R_GrassTextureForName( const char *name, byte *tex )
 {
-	byte i;
+	size_t i;
+
 	for( i = 0; i < GRASS_TEXTURES && grasstexs[i].name[0]; i++ )
 	{
 		if( !Q_stricmp( grasstexs[i].name, name ))
-			return i;	// found
+			goto end;	// found
```

Comment:
I think it's better to just copy a little piece of code again instead of using goto

Reply:
Why? It's more compact and looks readable.
Compiler will do the same anyway.

Reply:
![goto](https://github.com/SNMetamorph/PrimeXT/assets/25657591/a1f07f25-2b0c-4d75-bd1e-1d3016498717)

Reply:
My brother in Christ, you have C++, use RAII! :D 

just kiddin

Reply:
https://en.wikipedia.org/wiki/Considered_harmful